src/skagent/block.py

```diff
@@ -337,11 +337,35 @@ def display(self, calibration):
 
         calibration: dict
             A dictionary of parameters used for calibration. Here, it indicates which symbols are not dynamic.
+
+
+        Returns
+        -----------
+
+        img : matplotlib.image.mpimg
+        svg_content : str
         """
         graph = self.visualize(calibration)
 
         display(SVG(graph.create_svg()))
 
+        import matplotlib.image as mpimg
+        from io import BytesIO
+        import cairosvg
```

Comment:
are the imports here necessary? visualization should be delegated to a separate, centralized module